README.md

```diff
@@ -243,6 +249,72 @@ foreach ($scoreResponse->warnings as $warning) {
 }
 ```
 
+### Report Transactions API ###
+
+MaxMind encourages the use of this API as data received through this channel is
+used to continually improve the accuracy of our fraud detection algorithms.
+
+To use the Report Transactions API, create a new `\MaxMind\ReportTransaction`
```

Comment:
I didn't notice this before, but I wonder if `\MaxMind\MinFraud\ReportTransaction` would be a better name. It is a minFraud API, just not one of our scoring APIs.

Similarly, `\MaxMind\ServiceClients` could lead to accidental name collisions with our other client APIs.

Reply:
I've made this change.